app/assets/javascripts/admin/log_list_view.coffee

```diff
@@ -15,13 +15,20 @@ class MS.Views.LogListView extends Backbone.View
     @logFormModalView.showEdit(logId, '', @refresh.bind(@))
 
   deleteLog: (e, resp) ->
-    e.preventDefault()
     logId = @$(e.currentTarget).closest('.log').data('id')
+    context = @$(e.currentTarget).data('context')
     if (resp)
-      $.ajax(method: "DELETE", url: "/admin/logs/#{logId}")
-      .done => @refresh()
+      $.ajax(method: "DELETE", url: "/admin/logs/#{logId}/?context=#{context}")
+      .done (response) =>
+        # Replace log list
+        @$el.html(response)
+
+        # Remove log from timeline
```

Comment:
This is only needed if we are in the timeline context and the "More" link is not deleted if there are no more logs left for a specific step.

I started trying to pass a context to the step modal and then this would later pass the context to the logs list. However, the step modal did not render correctly.

I've stopped on that piece for now. Perhaps removing the "More" when no logs are left can be added to the backlog.